applications/luci-app-filemanager/htdocs/luci-static/resources/view/system/filemanager.js

```diff
@@ -0,0 +1,1571 @@
+                if (field) {
+                    self.sortBy(field);
+                }
+            });
+        });
+
+        this.loadFileList(currentPath).then(function() {
+            self.initResizableColumns();
+            // Добавляем наблюдатель за изменением размера окна
+            var fileListContainer = document.getElementById('file-list-container');
+            if (fileListContainer && typeof ResizeObserver !== 'undefined') {
+                var resizeObserver = new ResizeObserver(function(entries) {
+                    for (var entry of entries) {
+                        var newWidth = entry.contentRect.width;
+                        var newHeight = entry.contentRect.height;
+                        config.windowSizes.width = newWidth;
+                        config.windowSizes.height = newHeight;
+                    }
+                });
+                resizeObserver.observe(fileListContainer);
+            }
+        });
+        return viewContainer;
+    },
+switchToTab: function(tab) {
+    var fileManagerContent = document.getElementById('content-filemanager');
+    var editorContent = document.getElementById('content-editor');
+    var settingsContent = document.getElementById('content-settings');
+    var tabFileManager = document.getElementById('tab-filemanager');
+    var tabEditor = document.getElementById('tab-editor');
+    var tabSettings = document.getElementById('tab-settings');
+
+    if (fileManagerContent && editorContent && settingsContent && tabFileManager && tabEditor && tabSettings) {
+        fileManagerContent.style.display = (tab === 'filemanager') ? 'block' : 'none';
+        editorContent.style.display = (tab === 'editor') ? 'block' : 'none';
+        settingsContent.style.display = (tab === 'settings') ? 'block' : 'none';
+        tabFileManager.className = (tab === 'filemanager') ? 'cbi-tab cbi-tab-active' : 'cbi-tab';
+        tabEditor.className = (tab === 'editor') ? 'cbi-tab cbi-tab-active' : 'cbi-tab';
+        tabSettings.className = (tab === 'settings') ? 'cbi-tab cbi-tab-active' : 'cbi-tab';
+
+        if (tab === 'settings') {
+            this.loadSettings(); // Загружаем настройки при переходе на закладку
+        }
+    }
+},
+handleGoButtonClick: function() {
+    var self = this;
+    var pathInput = document.getElementById('path-input');
+    if (pathInput) {
+        var newPath = pathInput.value.trim() || '/';
+        // Проверяем, существует ли путь и является ли он директорией
+        fs.stat(newPath).then(function(stat) {
+            if (stat.type === 'directory') {
+                currentPath = newPath;
+                pathInput.value = currentPath;
+                self.loadFileList(currentPath).then(function() {
+                    self.initResizableColumns();
+                });
+            } else {
+                ui.addNotification(null, E('p', _('Указанный путь не является директорией.')), 'error');
```

Comment:
I'm not sure most users would bother with or be satisfied with russian. Something like `The specified path does not appear to be a directory`.

Reply:
Fixed. All comments and messages are in English now